### ISSUE TITLE
VINS-Mask algorithm

### ISSUE THREAD

**Issue by sheldon-yezi** [kminoda/VIODE#21]
I can‘t find the details of VINS-Mask algorithm in your paper, Is it open source or not?’ I find VINS-Mask in the github https://github.com/sunjiayuanro/VINS-Mask，is this your work？Looking forward to your reply

**Comment by kminoda** [comment]
No, it's not open sourced, but it's a very simple method, incorporating mask to filter out the feature points in VINS-Mono. And the code you've shared is not relevant to ours. Indeed, we were not aware of the project. Thank you for the information.
